content/get-haiku/r1beta4/release-notes.md

```diff
@@ -0,0 +1,189 @@
+
+### Support for (some) USB WiFi devices
+
+Most "RTL" and some "RA" USB WiFi devices (the two most common device families, very often repackaged under other names) are now supported in Haiku thanks to the import of the relevant drivers from FreeBSD, and the implementation of FreeBSD's basic USB-related APIs in Haiku's FreeBSD driver compatibility layer. There is one major limitation: these devices must be connected before Haiku is booted; they will not be detected if connected after the boot is finished.
+
+### WiFi drivers from OpenBSD
+
+The `iwm` and `iwx` drivers from OpenBSD, along with OpenBSD's 802.11 stack, have been imported into Haiku (and given the native names `idualwifi7260` and `iaxwifi200` respectively.) These provide support for Intel's "Dual Band" and "AX" families of WiFi hardware. FreeBSD's version of the `iwm` driver had been available on Haiku for years, but only supported 802.11a/b/g (not even 802.11n, much less 802.11ac!) and was not very reliable; meanwhile the `iwx` driver has no direct FreeBSD equivalent.
+
+This means Haiku is only the third open-source operating system (to the best of our knowledge) after Linux and OpenBSD to support 802.11ac WiFi, as not even FreeBSD supports it (yet?) despite having worked on it intermittently for years.
+
+Supporting these drivers required the addition of an "OpenBSD compatibility layer", which builds on top of the existing FreeBSD compatibility layer. OpenBSD supporting hardware that FreeBSD does not seems to be an increasing trend (there are some more ethernet and WiFi cards that OpenBSD has drivers for which Haiku could stand to gain), and this work paves the way for such future developments.
+
+### New NTFS driver
+
+The NTFS filesystem driver (which is a wrapper around the NTFS-3G library) was completely rewritten. The old one hearkened back to the BeOS days and had not kept up with development: it did not support any kind of caching, incorrectly handled file deletions in a way that caused kernel panics, did not support memory-mapping files (making `git` usage impossible on NTFS partitions), and had plenty of other issues besides.
+
+The new diver is much more idiomatic, supports the major layers of Haiku's common filesystem caching mechanisms, and resolves all known issues and major missing features the previous one was plagued with. Since it is not FUSE-based (like NTFS-3G is on Linux) but runs in kernel-mode, it is competitive for performance with the other fully native filesystem drivers.
+
+### AVIF translator
+
+There is now an AVIF image translator available by default, providing both read and write support for this new and advanced image format for all native Haiku applications.
+
+### USB-RNDIS tethering support
+
+A new driver has been added to support "RNDIS" ethernet tethering, which is the most common kind mobile phones use when selecting "USB tethering". It is enabled by default, and supports hot-plugging.
```

Comment:
It is common for Android phones, Apple uses something else and so do older phones which are usually ppp based